torch/_prims_common/__init__.py

```diff
@@ -241,140 +160,6 @@ def is_contiguous(a: TensorLikeType) -> bool:
     return True
 
 
-# This function is equivalent to compute_channels_last_contiguous_2d() in TensorImpl.cpp
-def is_channels_last_contiguous_2d(a: Tensor) -> bool:
```

Comment:
where did all these functions go?

Reply:
hrmm, I suspect a recent merge failure.

Reply:
oh ok, I think constant_pad_nd was reverted

Reply:
Fixed the issue.